### PR TITLE
[EX-3937] update files

### PR DESCRIPTION
**Pull request by pmareke**
-     Add LICENSE file to package copying it from the root folder
-     Create or update the README file
-     Check package.json to update information (homepage, author, license, description)
-     Check package.json to update dependencies
-     Check package.json to update scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+[![Created by](https://img.shields.io/badge/Created%20by-Empathy.co-green)](https://www.empathy.co)
+
 Welcome to the X Components Archetype
 
 ## What is this repository for?
 
-The purpose of this repository is to be the starting point to a customer setup using the X
-Components. Please take a look at the [documentation](./docs/index.md).
+The purpose of this repository is to be the starting point to a customer setup using the X Components. Please take a look at
+the [documentation](./docs/index.md).
```
Comment (tiborux) [inline README.md:11]: should we update the documentation? It is referring bitbucket

File: package.json [hunk 2]
```diff
@@ -31,7 +35,7 @@
   "devDependencies": {
     "@cypress/webpack-preprocessor": "~5.1.1",
     "@empathybroker/deep-merge": "^1.2.1",
-    "@empathy/eslint-plugin-x": "^1.2.0",
+    "@empathyco/eslint-plugin-x": "^1.2.0",
```
Comment (LuisMartinez15) [inline package.json:38]: Can you move this dependency below the `@empathy/x-translations` one? So it is ordered alphabetically ;)

File: README.md (hunk 1 above)
Comment (LuisMartinez15) [reply README.md:11]: We definitely should discuss about changes in the documentation. How is this project going to be used? How are clients setups/projects going to be handled/managed? Are they going to be public in github? Cause I don't think so... 

So, in the end, I think we should have a team discussion about this topic.

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (herrardo) [reply README.md:11]: Yes, we should have that discussion. The client repos are going to be in Github, but private of course. We should treat this repo as the default/preferred/mostloved/startingpoint search experience to be forked/duplicated to start an easy instrumentation.

**Review by LuisMartinez15 — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**